src/status_im/ui/components/list_item/views.cljs

```diff
@@ -340,19 +358,28 @@
 ;; disabled?
 ;; boolean - false by default - self explanatory
 
+;; selected?
+;; boolean
+;; when (= :theme :selectable) this switch controls whether the
+;; list-item is in a visually selected state (background-color
+;; of list-item is colors/gray-selected
+;; useful for selectable list-items like list with radio buttons
+
 (defn list-item
```

Comment:
could you move all these comments in the docstring if that seems relevant?

Reply:
yes indeed - forgot about docstring 👍

Reply:
done 👍 
followed this convention https://github.com/bbatsov/clojure-style-guide#documentation